amundsen_application/api/metadata/v0.py

```diff
@@ -81,6 +81,8 @@ def _map_results(result: Dict) -> Dict:
             response_list = response.json().get('popular_tables')
             top4 = response_list[0:min(len(response_list), POPULAR_TABLE_COUNT)]
             popular_tables = [_map_results(result) for result in top4]
+            for table in popular_tables:
+                table['type'] = 'table'
```

Comment:
can't we simply add this information directly in the `_map_results()` function instead of introducing a separate loop?